src/gen_cranberry/gc_dev_batt.cpp

```diff
@@ -10,6 +10,10 @@ uint16_t gc_dev_batt_read(void){
     uint16_t value = 4000;
 
     #ifndef SEN_STUB
+    /* Scaling factor of 2.0 is used because of the voltage divider
+    placed in the circuit (Note: Cranberry v3.5.0 Schematic incorrect 
+    R21 and R20 are described to have values of 150k and 51k respectively,
+    these values are the same in hardware. */
```

Comment:
```
/* Scaling factor of 2.0 is used because of the voltage divider
placed in the circuit (Note: Cranberry v3.5.0 Schematic incorrect 
R21 and R20 are described to have values of 150k and 51k respectively,
these values are the same in hardware. */
```

Looks okay, but I would change the wording to make it easier to read. Something like this:

```
/*
Multiply the result by 2.0 because the ADC is connected to a voltage divider circuit
that halves the original battery voltage.

Note: the cranberry v3.5.0 schematic is incorrect because the values for R21 and 
R20 are described with values of 150k and 51k respectively. In reality, R21 and
R20 are equal to each other (the values are still unkown as of 2016-10-24).
*/
```